cpp/tests/transform/integration/unary_transform_test.cpp

```diff
@@ -491,4 +568,67 @@ TEST_F(StringOperationTest, MixedTypes)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view());
 }
 
+TEST_F(StringOperationTest, Output)
+{
+  auto a = cudf::test::strings_column_wrapper{"this", "b", "c", "d", "e", "f"};
+  auto b = cudf::test::strings_column_wrapper{"aa", "is", "dd", "ddd", "e", "fff"};
+  auto c = cudf::test::strings_column_wrapper{"a", "b", "the", "dddd", "e", "fff"};
+  auto d = cudf::test::strings_column_wrapper{"a", "b", "d", "largest", "lexicographical", "test"};
+
+  std::string cuda = R"***(
+    __device__ void transform(cudf::string_view * out, cudf::string_view a, cudf::string_view b, cudf::string_view c, cudf::string_view d){
+      *out =  std::max(std::max(std::max(a, b), c), d);
+    }
+    )***";
+
+  auto expected =
+    cudf::test::strings_column_wrapper{"this", "is", "the", "largest", "lexicographical", "test"};
+  auto result = cudf::transform({a, b, c, d}, cuda, cudf::data_type(cudf::type_id::STRING), false);
+
+  CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view());
+}
+
+TEST_F(StringOperationTest, StringConcat)
+{
+  auto first_name =
+    cudf::test::strings_column_wrapper{"John", "Mia", "Abd", "Mendes", "Arya", "John"};
```

Comment:
Should we test non-ASCII characters someplace in this file? We are typically pretty rigorous about including non-ASCII characters to ensure we have correct coverage.

Reply:
We could, but it wouldn't be of much value given it's left to the UDF to handle the logic. 
We'd be testing the UDF instead of testing the transform. I'll add some utf-8 names to test either way.